src/components/Fund/FundsList.js

```diff
@@ -134,16 +141,9 @@ class FundsList extends Component {
 
   render() {
     const { onSelectRow, onComponentWillUnmount, resources, mutator, match, stripes } = this.props;
+    const ledgers = get(resources, ['ledgers', 'records'], []);
     const resultsFormatter = {
-      'name': data => get(data, ['name'], ''),
-      'code': data => get(data, ['code'], ''),
-      'fundStatus': data => get(data, ['fundStatus'], ''),
-    };
-
-    const columnMapping = {
-      'name': 'Name',
-      'code': 'Code',
-      'fundStatus': 'Fund Status',
+      ledger: ({ ledgerId }) => get(ledgers.find(({ id }) => id === ledgerId), 'name', ''),
```

Comment:
I think this could be improved by breaking on 2 ops: 1. get map of ledgers (`_.keyBy(arr, 'id')`) 2. access that map in formatter, 
instead of lurking the whole array every time

Reply:
Done